rfcs/text/0010-email.md

```diff
@@ -1,7 +1,7 @@
 # 0010: Email
 <!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
 
-- Stage: **1 (proposal)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Stage: **2 (proposal)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
```

Comment:
```suggestion
- Stage: **1 (draft)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
```

As my earlier [comment](https://github.com/elastic/ecs/pull/1219#issuecomment-768588593) mentions, this is only to align with the new, condensed stages. Moving to `Draft` will still add the `email.*` fields to ECS as experimental fields.